web/client/plugins/TOC/components/WMSLegend.jsx

```diff
@@ -92,6 +96,35 @@ class WMSLegend extends React.Component {
                     />
                 </div>
             );
+        } else if (showLegend) {
```

Comment:
else if it's not needed if you are using an early return, you can use directly if (showLegend)